pkg/volume/iscsi/iscsi_util.go

```diff
@@ -126,6 +126,13 @@ func updateISCSINode(b iscsiDiskMounter, tp string) error {
 			}
 		}
 	}
+
+	// setting node.session.scan to manual to handle https://github.com/kubernetes/kubernetes/issues/90982
+	out, err = execWithLog(b, "iscsiadm", "-m", "node", "-p", tp, "-T", b.Iqn, "-I", b.Iface, "-o", "update", "-n", "node.session.scan", "-v", "manual")
+	if err != nil {
+		// don't fail if iscsiadm fails or the version does not support node.session.scan - log a warning to highlight the potential exposure
+		klog.Warningf("iscsi: failed to update node with node.session.scan=manual, possible exposure to issue 90982: %v", out)
+	}
```

Comment:
Now that I look at this again, if they're not using CHAP `!b.chap_session`, it will skip out before executing this which is not what we want. @jsafrane @bswartz @humblec - do you think keep it in updateISCSINode, before `!b.chap_session` or move it outside, right before `updateISCSINode` https://github.com/kubernetes/kubernetes/blob/release-1.18/pkg/volume/iscsi/iscsi_util.go#L345

Reply:
I'd move it on top of `updateISCSINode`, before `!b.chap_session`.